array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,22 @@
+/* exported includes */
+// input 1: array
+// input 2: value to search for
+// output: boolean indicating the presence of value at any index of array
+// if the array is empty
+// return false
+// look over each item
+// if the current item is the same as the input value
+// return true
+// return false for the remaining values
+
+function includes(array, value) {
+  if (array.length === 0) {
+    return false;
+  }
```

Comment:
This conditional isn't necessary. If there is no length to the array, the loop will never start, and you will return `false` underneath it, leaving you with the same result 🤓